devicetypes/tommysqueak/danfoss-living-connect-radiator-thermostat.src/danfoss-living-connect-radiator-thermostat.groovy

```diff
@@ -13,17 +13,18 @@
  *  for the specific language governing permissions and limitations under the License.
  *
  */
+
 metadata {
-	definition (name: "Danfoss Living Connect Radiator Thermostat LC-13 v3", namespace: "tommysqueak", author: "Tom Philip") {
+	definition (name: "Danfoss Living Connect Radiator Thermostat LC-13 v3.01", namespace: "tommysqueak", author: "Tom Philip") {
 		capability "Actuator"
 		capability "Sensor"
 		capability "Thermostat"
 		capability "Battery"
 		capability "Configuration"
 		capability "Switch"
 
-		command "temperatureUp"
-		command "temperatureDown"
+		command "tempUp"
```

Comment:
I would've passed comments on that example, if it was in a PR ;) In general is better to be more explicit to avoid any misinterpretation. I think one of things that's important in this code vs the example, is that this device handler is controlling a sleepy device. So `tempUp` could be mean 'temporary up' (until the device awakes and then takes on the new temp).